contracts/sdk/indexer/smokeTest.ts

```diff
@@ -1,6 +1,7 @@
 import { Program, web3 } from "@project-serum/anchor";
 import fetch from "node-fetch";
-import { getMerkleRollAccountSize, Gummyroll } from "../gummyroll";
+import { Gummyroll } from "../../target/types/gummyroll";
```

Comment:
Actually, we should start stripping all of these Anchor types from our typescript codebase.

They were only used before we had Solita.

Reply:
We will do this in another PR